parsl/dataflow/dflow.py

```diff
@@ -191,6 +191,9 @@ def _create_task_log_info(self, task_id, fail_mode=None):
         task_log_info['task_stdin'] = self.tasks[task_id]['kwargs'].get('stdin', None)
         task_log_info['task_stdout'] = self.tasks[task_id]['kwargs'].get('stdout', None)
         task_log_info['task_stderr'] = self.tasks[task_id]['kwargs'].get('stderr', None)
+        task_log_info['task_fail_history'] = None
+        if self.tasks[task_id]['fail_history'] is not None:
+            task_log_info['task_fail_history'] = "; ".join(self.tasks[task_id]['fail_history'])
```

Comment:
It would be more consistent with`task_depends` to use `,` as a separator. Maybe fix that in a follow up PR.

Reply:
Okay. I changed that.